android/settings.gradle

```diff
@@ -0,0 +1,15 @@
+rootProject.name = 'RNExampleApp'
+apply from: file("../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesSettingsGradle(settings)
+
+// include ':onewelcome-react-native-sdk'
+// project(':onewelcome-react-native-sdk').projectDir = new File(rootProject.projectDir, '../../android')
```

Comment:
please remove the commented part or add TODO comment if this is something to be changed later